File: Java/Easy/FizzBuzz.java
class Solution {
    public List<String> fizzBuzz(int n) {
        if(n <= 0)
            return new ArrayList();
        TreeMap<Integer, String> wordMap = new TreeMap<>((a, b) -> a - b);
        wordMap.put(3, "Fizz");
        wordMap.put(5, "Buzz");
        
        List<String> thisStrs = new LinkedList();
        
        for(int i = 1; i <= n; i++){
            
            StringBuilder fizzBuzz = new StringBuilder();
            
            for(int number: wordMap.keySet()){
                if(i % number == 0)
                    fizzBuzz.append(wordMap.get(number));
            }
            
            thisStrs.add((fizzBuzz.length() == 0) ? String.valueOf(i) : fizzBuzz.toString());
        }
        
        return thisStrs;
    }
}

File: Sprint/FizzBuzz.py
def fizzBuzz(self, n: int) -> List[str]:
        ans = []
        for i in range(1, n + 1):
            this_s = ''
            
            if i % 3 == 0:
                this_s += 'Fizz'
            
            if i % 5 == 0:
                this_s += 'Buzz'
            
            ans.append(this_s if this_s != '' else str(i))
        
        return ans
